website/src/pages/index.tsx

```diff
@@ -138,12 +138,16 @@ function Hero() {
               }}
             </BrowserOnly>
           </div>
-          <div className="flex justify-center">
-            <img
-              className="md:w-5/6 lg:w-11/12 w-full"
-              src="https://raw.githubusercontent.com/containers/podman-desktop/media/screenshot.png"
```

Comment:
Hello, could you explain why you want to duplicate the image locally ?

Reply:
Security :) better to have it locally rather than a raw link that may change! With GitHub down this morning, pages were working, but no raw links, so it actually appeared blank.

Reply:
if you prefer one option over the other one, we should kill the screenshot.png from the main branch and use the image from the website (using relative path to website folder) inside the README
https://github.com/containers/podman-desktop/blob/main/README.md?plain=1#L5